lib/cover_my_meds/api_request.rb

```diff
@@ -25,8 +25,14 @@ def request(http_method, host, path, params={}, auth_type = :basic, &block)
       rest_resource = RestClient::Resource.new(uri.to_s, tail)
 
       response = call_api http_method, rest_resource, &block
+      parse_response response
```

Comment:
Not calling a method should be equivalent to nil, in the sense that the return value of a method will be nil if a method doesn't get called on its last line (as in the above case).  I tried it at IRB and it did work out that way.

Seeing as you've already merged it's not a huge deal though :)